File: strings/testCat/strcat.py
import time

def strcat():
    str1 = input("Insira o arquivo para concatenar a string: ")
    str2 = input("Insira as strings a serem concatenadas: ")
    
    t1 = time.time()
    for i in range(0, len(str2)):
        str1[i] = str1[i] + str2[i];
    t2 = time.time()
    print(t2 - t1)

File: strings/testCat/strcat.java
import java.util.Scanner;

public class strcat {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str1;
        String str2;
        str1 = sc.nextLine();
        str2 = sc.nextLine();
            long t1 = System.currentTimeMillis();
                str2 = str1 + str2;
                System.out.println(str2);
            long t2 = System.currentTimeMillis();
            long t3 = t2 - t1;
            System.out.println(t3);
        sc.close();
    }
}